### PR TITLE
spread: disable general/chisel

### PR DESCRIPTION
**Pull request by tigarmo**
The spread test uses dotnet6 slices, but an update to dotnet-host broke the chisel cut operation. We can revert this commit once chisel itself is updated with the fix.

- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

**Review by mr-cal — APPROVED** [review]
Putting the PR's description into the task.yaml as a comment would be helpful.

**Comment by sergiusens** [comment]
I wouldn't disable this just yet. Give it a day.
-------- Original message --------From: Callahan ***@***.***> Date: 2022-11-29  10:11  (GMT-03:00) To: canonical/rockcraft ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [canonical/rockcraft] spread: disable general/chisel (PR #152) 
@mr-cal approved this pull request.

Putting the PR's description into the task.yaml as a comment would be helpful.

**Comment by tigarmo** [comment]
@sergiusens I updated the spread test to *pass* if `rockcraft pack` fails because of the Chisel bug. Should be simple enough to revert once the issue is fixed, and the constant spread failures were already giving me anxiety :laughing: 
I verified locally that the test fails if the log doesn't have that exact error message in it.

File: tests/spread/general/chisel/task.yaml [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+    echo "rockcraft pack succeeded unexpectedly!"
+    exit 1
+  else
+    # Find the "latest" log file to parse the error message
+    log_file=$(find /root/.cache/rockcraft/log/ -iname "rockcraft*.log" | sort -r | head -n1)
+
+    # "rockcraft pack" must only have failed because of the Chisel bug
+    expected_error="Command '\['chisel', 'cut', '--root', '/root/parts/chisel-part/install', 'dotnet-runtime-6.0_libs'\]' returned non-zero exit status 1"
+
+    grep "$expected_error" $log_file
```
Comment (sergiusens) [inline tests/spread/general/chisel/task.yaml:20]: can you use spread's MATCH here?

**Comment by cjdcordeiro** [comment]
no need anymore: https://github.com/canonical/chisel-releases/pull/23#pullrequestreview-1200572540

**Comment by tigarmo** [comment]
Nice that the dotnet issue was fixed, but the OpenSSL one is still there, right? This one: https://github.com/canonical/chisel/pull/36

**Comment by tigarmo** [comment]
Oh `chisel-releases` was updated, got it
Thanks!

**Comment by tigarmo** [comment]
Looks like the rockcraft.yaml file itself now needs to be updated, doing that on this other PR: https://github.com/canonical/rockcraft/pull/155

**Pull request closed without merging**